native-alerts/readme.md

```diff
@@ -1,12 +1,10 @@
 # Native Alerts Integration
 
-Native support in ThousandEyes for sending alerts to AppDynamics is currently on the product roadmap for Q1 2020 (Sept).
+AppDynamics alerts can be triggered by external sources like ThousandEyes using the AppDynamics Alert & Respond API. This API allows generating alert events in AppDynamics that can trigger alert policies and actions as well as overlay alerts in dashboards. The AppDynamics documentation refers to this API as “Alert and Respond API,” as well as “Events and Action API”; we’ll refer to it as the AppDynamics Alerts API to avoid confusion.
 
-AppDynamics alerts can be triggered by external sources like ThousandEyes using the AppDynamics Alert and Respond API. This API allows generating alert events in AppD that can trigger alert policies and actions as well as overlay alerts in dashboards. The AppD documentation refers to this API as “Alert and Respond API”, as well as “Events and Action API”; we’ll refer to it as the AppD Alerts API to avoid confusion.
+You can set up alert notifications using the AppDynamics notification drop-down in ThousandEyes. 
 
-Users can setup alert notifications using the AppD notification drop-down in ThousandEyes. 
+Once the notification is set up, you can associate the notification with any alert rules they have set up in ThousandEyes. Once this is done, ThousandEyes will send full alert data to AppDynamics whenever the associated alert is triggered. <!-- Here is an example of how alerts appear in AppDynamics: -->
```

Comment:
Remove this since it's covered in official docs.

Reply:
Which part do you want to remove? All of line 5?

Reply:
Yep all of line five.